Services/Meet/PsychologicalSupportPlatform.Meet.API/Controllers/ScheduleCellsController.cs

```diff
@@ -28,8 +28,7 @@ public ScheduleCellsController(IMediator mediator, IMapper mapper)
         [Authorize(Roles = Roles.Admin + "," + Roles.Psychologist)]
         public async Task<IActionResult> CreateScheduleCell(CreateScheduleCellDTO scheduleCell)
         {
-            var createCmdScheduleCell = _mapper.Map<AddScheduleCellDTO>(scheduleCell);
-            var command = _mapper.Map<CreateScheduleCellCommand>(createCmdScheduleCell);
+            var command = _mapper.Map<CreateScheduleCellCommand>(scheduleCell);
```

Comment:
In your commands, you can accept DTO models to eliminate unnecessary mapping in the controllers

Reply:
fixed